Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.12.5 as builder
```

Comment:
is it possible to build with 1.13.1? so we're up to date

Reply:
:+1: I think 1.13.1 is the latest now, makes sense to start there!

Reply:
Okay, I'll upgrade this to 1.13.1.